src/components/BuildConfigsList/BuildConfigsList.tsx

```diff
@@ -0,0 +1,254 @@
+
+  return (
+    <>
+      <Toolbar>
+        <ToolbarItem>
+          <Filtering
+            filterOptions={getFilterAttributes(buildConfigEntityAttributes, defaultColumns)}
+            componentId={componentId}
+            defaultFiltering={defaultFiltering}
+          />
+        </ToolbarItem>
+      </Toolbar>
+
+      <ContentBox borderTop>
+        <ServiceContainerLoading {...serviceContainerBuildConfigs} title={PageTitles.buildConfigs}>
+          <TableComposable isStriped variant="compact">
+            <Thead>
+              {/**
+               * If column order is changed, the property tableColumnIndex (see sortAttributes) has to be updated.
+               * Better solution can be implemented in the future.
+               */}
+              <Tr>
+                {columns.includes(buildConfigEntityAttributes.name.id) && (
+                  <Th width={15} sort={getSortParams(sortAttributes.name.id)}>
+                    {buildConfigEntityAttributes.name.title}
+                  </Th>
+                )}
+                {columns.includes(buildConfigEntityAttributes.description.id) && (
+                  <Th width={15}>{buildConfigEntityAttributes.description.title}</Th>
+                )}
+                {columns.includes(buildConfigEntityAttributes.buildType.id) && (
+                  <Th width={10} sort={getSortParams(sortAttributes.buildType.id)}>
+                    {buildConfigEntityAttributes.buildType.title}
+                  </Th>
+                )}
+                {columns.includes(buildConfigEntityAttributes['project.name'].id) && (
+                  <Th width={15} sort={getSortParams(sortAttributes['project.name'].id)}>
+                    {buildConfigEntityAttributes['project.name'].title}
+                  </Th>
+                )}
+                {(columns.includes(buildConfigEntityAttributes.creationTime.id) ||
+                  columns.includes(buildConfigEntityAttributes.modificationTime.id)) && (
+                  <Th width={20} className="overflow-visible">
+                    <SortGroup
+                      title="Times"
+                      sort={getSortGroupParams(
+                        columns.includes(buildConfigEntityAttributes.creationTime.id)
+                          ? sortAttributes.creationTime.id!
+                          : sortAttributes.modificationTime.id!
+                      )}
+                      isDropdownOpen={isTimesSortDropdownOpen}
+                      onDropdownToggle={() => setIsTimesSortDropdownOpen(!isTimesSortDropdownOpen)}
+                    />
+                  </Th>
+                )}
+                {columns.includes(buildConfigEntityAttributes.buildStatus.id) && (
+                  <Th width={25}>{buildConfigEntityAttributes.buildStatus.title}</Th>
+                )}
+                {columns.includes(buildConfigEntityAttributes.actions.id) && (
+                  <ProtectedComponent hide>
```

Comment:
@DnsZhou can you please create new for ProtectedContent?

Reply:
NCL-8095 created for this issue

Reply:
The action won't pass to the backend anyway when not logged in, because the backend will return a 401 error. we can keep it like this to automatically fix it when NCL-8095 finishes.